### DISCUSSION TITLE
IgnoreMap and Ignore Attributes are different

### DISCUSSION THREAD

**Discussion by schmitch** [LuckyPennySoftware/AutoMapper#3946] [NA]
### Source/destination types

```csharp

/* we used automapper to map to the same type but ignore some fields, to remove duplicate mappings in efcore when an include happened, long story.... */ 
public class X
{
    public string XT { get; set; }
}

public class Hase
{
    public int Id { get; set; }
    // 11.0 does not work 11.0: [Ignore]
    // 10.1 does work: [IgnoreMap]
    public X X { get; set; }
}
```

### Mapping configuration

```csharp

var mapper = new Mapper(new MapperConfiguration(config =>
{
    config.CreateMap<Hase, Hase>();// Works on both Versions: .ForMember(h => h.X, f => f.Ignore());
}));


```

### Version: x.y.z
10.1.1
11.0.1

### Expected behavior

the `Ignore` Annotation should ignore the ignored member

### Actual behavior

currently the `Ignore` Annotation did not ignore the member

### Steps to reproduce

```csharp
Console.WriteLine(JsonSerializer.Serialize(x));
Console.WriteLine(JsonSerializer.Serialize(mapper.Map<Hase>(x)));
```

well we can easily create a manual mapper configuration which does work, unfortunatly the annotations are a little bit different than before (but I see no reason why Ignore should not Ignore the field tough)

**Comment by lbargaoanu** [comment]
https://docs.automapper.org/en/latest/11.0-Upgrade-Guide.html#maptoattribute-and-ignoremapattribute-were-removed
`[Ignore]` only applies to attribute mapping.
